src/client/app/actions/groups.js

```diff
@@ -354,6 +365,11 @@ export function deleteGroup() {
 					dispatch2(changeDisplayMode('view'));
 				});
 			})
-			.catch(console.error);
+			.catch(dispatch(showNotification({
+				message: 'Failed to delete group',
+				level: 'error',
+				position: 'tr',
+				autoDismiss: 3
+			})));
```

Comment:
It's fine to show a notification, but it should also log to the console.
This is also the third usage of this exact logic in this file. It should be factored out into a function of its own.